mlir/tools/mlir-rocm-runner/mlir-rocm-runner.cpp

```diff
@@ -51,6 +51,7 @@ void registerTestDialect(DialectRegistry &);
 } // namespace test
 
 static LogicalResult runMLIRPasses(ModuleOp m) {
+  m.getContext()->disableMultithreading();
```

Comment:
What was the problem that caused the need to disable multi threading? I thought we already put a lock on the linker?

Reply:
Actually I don't think that is necessary. This was an attempt to fix the hang which turned out to be a node problem. I will revert this.

Reply:
Sounds good, hopefully no issue show up when we revert it.